layouts/partials/embedshare.haml

```diff
@@ -0,0 +1,57 @@
+%h3 Embed into your Website
+%input{:type => "text", :size=> "85", :readonly => true, 
+       :value => "<iframe width=\"853\" height=\"480\" src=\"#{oembed_page_url(@item.identifier)}\" frameborder=\"0\" allowfullscreen></iframe>"}
+%h3 Share with others
+%ul
+  %li
+    %a{:id => "btnTweet", :href => "#"}
```

Comment:
The A-Tags should link directly to the sharing url (so that they work without js). The javascript would then just call window.open(this.href, ...)